openupgrade_scripts/scripts/event/18.0.1.9/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,64 @@
+---Models in module 'event'---
+model event.question (moved from website_event)
+model event.question.answer (moved from website_event)
+model event.registration.answer (moved from website_event)
+---Fields in module 'event'---
+event        / event.event              / badge_format (selection)      : selection_keys is now '['96x134', '96x82', 'A4_french_fold', 'A6', 'four_per_sheet']' ('['A4_french_fold', 'A6', 'four_per_sheet']')
+event        / event.event              / general_question_ids (one2many): previously in module website_event
+event        / event.event              / question_ids (one2many)       : previously in module website_event
+event        / event.event              / specific_question_ids (one2many): previously in module website_event
+event        / event.mail               / last_registration_id (many2one): NEW relation: event.registration
+event        / event.mail               / notification_type (selection) : not stored anymore
+event        / event.mail               / notification_type (selection) : now a function
```

Comment:
group with event.type.mail  notification_type and put `nothing to do: https://github.com/odoo/odoo/commit/4b8576b49d8eec8260e276e06b75429bd9500f8c`